src/main/java/formflow/library/config/ScreenNavigationConfiguration.java

```diff
@@ -21,18 +21,57 @@ public class ScreenNavigationConfiguration {
   private String subflow;
   //TODO: Implement callback
   private String callback;
-  private String beforeSave;
```

Comment:
Doesn't _seem_ like we need to keep this string variable after we've setup the `Action`

Reply:
Yeah, I'd agree, but we are stuck with this because of how we load this class. We use Snake YAML and it overlays yaml onto the class.  Did this actually work with out it?   Maybe it just needs the setter.... hmmm...

Reply:
Yup :) - Works without it and just needs the setter

Reply:
Nice - I'm taking these out of the other PR as well.